src/bundle/Resources/public/scss/_main-menu.scss

```diff
@@ -117,6 +128,19 @@
             top: calculateRem(12px);
             left: calculateRem(48px);
         }
+
+        &--group-name {
+            color: $ibexa-color-light-700;
+            font-size: $ibexa-text-font-size-small;
+
+            &:before {
```

Comment:
Why to add the border-top you have to use `before`? Should be added on the element itself.

Reply:
it was a fix for border-top animation on menu toggler, I will remove it